widget/ColorPicker.js

```diff
@@ -521,8 +524,13 @@ define([
 			//	evt.preventDefault();
 			var satSelCenterH = this.PICKER_SAT_SELECTOR_H/2;
 			var satSelCenterW = this.PICKER_SAT_SELECTOR_W/2;
-			var newTop = evt.layerY - satSelCenterH;
-			var newLeft = evt.layerX - satSelCenterW;
+			
+			//#13268 Fix for IE and Edge, as they don't support evt.layerX/Y
+			
+			var newTop = evt.layerY || (evt.y -evt.target.getBoundingClientRect().top);
+			newTop-= satSelCenterH;
+            var newLeft = evt.layerX || (evt.x -evt.target.getBoundingClientRect().left);
```

Comment:
These two lines appear to use spaces instead of tabs, also please add a space between - and evt.target...